services/oauth/src/main/java/org/mule/services/oauth/internal/AbstractOAuthDancer.java

```diff
@@ -238,7 +223,7 @@ protected TokenResponse invokeTokenUrl(String tokenUrl, Map<String, String> toke
    * @param resourceOwnerId id of the user.
    * @return oauth state
    */
-  protected DefaultResourceOwnerOAuthContext getContextForResourceOwner(final String resourceOwnerId) {
+  public ResourceOwnerOAuthContext getContextForResourceOwner(final String resourceOwnerId) {
```

Comment:
Why is this in the abstract class if only authorisation code needs it according to it's api?